docs/src/pages/docs/functions/combinators.md

```diff
@@ -62,63 +144,85 @@ each branching function, the results of which will be applied to the accumulatin
 function.
 
 ```javascript
-import Maybe from 'crocks/Maybe'
-import alt from 'crocks/pointfree/alt'
 import converge from 'crocks/combinators/converge'
-import prop from 'crocks/Maybe/prop'
 
-const { Just } = Maybe
+import alt from 'crocks/pointfree/alt'
+import prop from 'crocks/Maybe/prop'
+import propOr from 'crocks/Maybe/propOr'
+import liftA2 from 'crocks/helpers/liftA2'
 
 // data :: [ Number ]
 const data = [ 1, 2, 3, 4, 5 ]
 
 // divide :: Number -> Number -> Number
-const divide = x => y => x / y
+const divide = x => y =>
+  y / x
+
+// add :: Number -> Number -> Number
+const add = a => b =>
+  b + a
 
 // sum :: [ Number ] -> Number
-const sum = xs => xs.reduce((m, n) => m + n, 0)
+const sum = xs =>
+  xs.reduce(add, 0)
 
 // length :: [ a ] -> Number
-const length = xs => xs.length
+const length =
+  propOr(0, 'length')
 
 // average :: [ Number ] -> Number
-const average = converge(divide, sum, length)
+const average =
+  converge(divide, length, sum)
 
 average(data)
```

Comment:
`add` needs to either be `curried` or `const add => (x, y} => x + y`. because this is coming up `NaN` when it tries to divide with as function